addon/components/nrg-appbar/component.js

```diff
@@ -19,8 +20,17 @@ export default Component.extend(ResizeMixin, {
   environmentDisplay: computed(
     'applicationSettings.localEnvironment',
     function() {
+      const ENV = getOwner(this).resolveRegistration('config:environment')[
+        'ember-nrg-ui'
+      ];
+      let productionEnvironments;
+      if (ENV) {
+        productionEnvironments = ENV.productionEnvironments || ['prod'];
+      } else {
+        productionEnvironments = ['prod'];
+      }
```

Comment:
This can be simplified some.

```js
const ENV = getOwner(this).resolveRegistration('config:environment');
const config = ENV['ember-nrg-ui'];
const productionEnvironments = config && config.productionEnvironments || ['prod'];
```